llvm/test/tools/llvm-readobj/ELF/dynamic-tags.test

```diff
@@ -138,6 +140,314 @@
 # GNU64-NEXT:  0x0000000076543210 (<unknown:>0x76543210) 0x5555666677778888
 # GNU64-NEXT:  0x0000000000000000 (NULL)                 0x0
 
+# JSON64:   "DynamicSection": [
```

Comment:
```suggestion
#      JSON64:   "DynamicSection": [
```
Do this and you still get the output nicely lined up for ease of reading.

That being said, I'd expect no spaces before "DynamicSection". IIRC, `--strict-whitespace` doesn't check leading and trailing whitespace without `--match-full-lines`, so I'm not sure there's a huge amount of point to having the former without the latter in this context (note that this is what the other cases do too). I'm also not 100% sure if it matters or not really, but I don't think it hurts to check them.

Reply:
I removed the spaces before "DynamicSection" and lined up that first line

Reply:
Once we are good with the format I'll clean up the commit message so the output matches @ilovepi to keep everything tidy